package/kedro_viz/api/rest/router.py

```diff
@@ -27,7 +28,7 @@
 )
 
 
-@router.get("/main", response_model=GraphAPIResponse)
+@router.get("/main", response_class=EnhancedORJSONResponse)
```

Comment:
```suggestion
@router.get("/main", response_class=EnhancedORJSONResponse, response_model=GraphAPIResponse)
```

Don't we still want the `response_model`?

Reply:
You're right, fixed in [207a718](https://github.com/kedro-org/kedro-viz/pull/1087/commits/207a7181087acc9439056c47fc870f88ce3ae76d).